test/functional/wallet_multiwallet.py

```diff
@@ -211,5 +211,15 @@ def run_test(self):
         # Fail to load if wallet file is a symlink
         assert_raises_rpc_error(-4, "Wallet file verification failed: Invalid -wallet path 'w8_symlink'", self.nodes[0].loadwallet, 'w8_symlink')
 
+        self.log.info("Test dynamic wallet creation.")
+
```

Comment:
Could test invalid argument.

Reply:
We don't exhaustively test invalid args for RPCs (since it's just hitting the univalue type checking code).